freshmaker/handlers/__init__.py

```diff
@@ -582,6 +586,7 @@ def build_image_artifact_build(self, build, repo_urls=None):
         return self.build_container(
             scm_url, branch, target,
             repo_urls=repo_urls,
+            flatpak=flatpak,
             isolated=True,
```

Comment:
By checking some old flatpak runtime images, seems these flatpak images need to be built with "isolated=False".

Reply:
OK, I added support for fetching the "isolated" parameter from build.